Assets/Mirror/Runtime/Transport/Websocket/WebsocketTransport.cs

```diff
@@ -22,6 +23,10 @@ public class WebsocketTransport : Transport
         [Tooltip("Nagle Algorithm can be disabled by enabling NoDelay")]
         public bool NoDelay = true;
 
+        [Tooltip("How long between each ping pong message before we disconnect end users Set to 0 to disable. Note" +
+                 " if server is overwhelmed or slammed with messages may need to increase this timer or disable altogether.")]
+        public int KeepAliveTimer = 300;
```

Comment:
```suggestion
        public int KeepAliveTimer = 30;
```

Reply:
We cannot set this to 30. Its already at 30 and causing major issues this is why i suggested the minimal 300 and have it adjustable from there. Setting this to 30 will just have more users messaging in discord while they are having major disconnects.